File: frontend/config-loader.js
const fs = require('fs');
const path = require('path');

const CONFIG_DIR = path.join(__dirname, '..', 'config');
const ENV = process.env.ENV || 'dev';
const CONFIG_PATH = path.join(CONFIG_DIR, `${ENV}.yaml`);

const yaml = require('js-yaml');

function loadConfig() {
  return yaml.load(fs.readFileSync(CONFIG_PATH, 'utf8'));
}

module.exports = { loadConfig };

File: api/jules_api/config_loader.py
import os
import yaml
from pathlib import Path

CONFIG_DIR = Path('/config')

ENV = os.environ.get('ENV', 'dev')

CONFIG_PATH = os.path.join(CONFIG_DIR, f'{ENV}.yaml')

def load_config():
    with open(CONFIG_PATH, 'r') as f:
        return yaml.safe_load(f)
